discord/no-decay/index.html

```diff
@@ -337,9 +277,8 @@ <h1 class="title is-1">Meet the staff<i>!</i></h1>
                                                                     </div>
                                                                 </div>
                                                                 <div class="content">
-                                                                    <p class="subtitle is-6 has-text-dark"><b>SR:</b> 2500</p>
                                                                     <p>
-                                                                        Owner and creator of the server Jelly is Mum. She keeps the staff in check and oversees everything running smoothly. Flexes for Team T2raining Bots.
```

Comment:
I guess this comment is a bit too tongue and cheek, yeah? 😛

Reply:
Maybe a little bit! So Jelly I don't think would care about being called "mum" but I was perhaps thinking of how that would look for prospective people looking to join the server